Aquality.Selenium.Template/Aquality.Selenium.Template.NUnit/Steps/ContactUsPageSteps.cs

```diff
@@ -93,7 +90,7 @@ public void CheckThatWarningEmailMessageisPresentOrNot(bool isChecked = false)
             AqualityServices.ConditionalWait.WaitForTrue(() => {
                 return contactUsPage.IsWarningEmailMessagePresent == isChecked;
             },
-             TimeSpan.FromSeconds(ProjectConstants.Timeout), TimeSpan.FromSeconds(ProjectConstants.PollingInterval),
+             TimeSpan.FromSeconds(1), TimeSpan.FromSeconds(5),
```

Comment:
discussed, to be replaced with e.g. ScriptTimeout and PollingInterval from the timeout configuration class + fix the order

Reply:
This code was updated.